core/src/main/java/io/questdb/cairo/IntervalBwdDataFrameCursor.java

```diff
@@ -76,12 +76,12 @@ public DataFrame next() {
 
                 // calculate intersection
 
-                long lo = search(column, intervalLo, 0, partitionLimit == -1 ? rowCount : partitionLimit);
+                long lo = search(column, intervalLo, 0, partitionLimit == -1 ? rowCount : partitionLimit, true);
```

Comment:
may be here we can pass increment itself as constant, e.g.:

```java
static final int SCAN_UP = -1;
static final int SCAN_DOWN = 1;

...
long lo = search(column, intervalLo, 0, partitionLimit == -1 ? rowCount : partitionLimit, SCAN_UP);
```
this is both more readable and faster :)

Reply:
Fixed.